package.json

```diff
@@ -21,5 +21,8 @@
     "tailwindcss": "^3.4.3",
     "typescript": "^5.4.5"
   },
-  "license": "GPL-3.0-or-later"
-}
\ No newline at end of file
+  "license": "GPL-3.0-or-later",
+  "devDependencies": {
+    "@tailwindcss/typography": "^0.5.13"
```

Comment:
This is going to cause the npm deps hash to change. Would you be able to update the nix package to have the new hash?